packages/checkout/widgets-lib/src/widgets/bridge/views/ClaimWithdrawal.tsx

```diff
@@ -169,13 +190,12 @@ export function ClaimWithdrawal({ transaction }: ClaimWithdrawalProps) {
 
     // send transaction to wallet for signing
     try {
-      // TODO: WT-2054 Update view to go to in progress screens and pass through sendTransaction response
-
-      // const response = await checkout.sendTransaction({
-      //   provider: providerToUse,
-      //   transaction: withdrawalResponse.unsignedTx,
-      // });
-
+      const response = await checkout.sendTransaction({
+        provider: providerToUse,
+        transaction: withdrawalResponse.unsignedTx,
+      });
+      // eslint-disable-next-line no-console
+      console.log(response);
```

Comment:
is this console.log intentionally committed?

Reply:
I think this is outdated now. Should be removed